cardano-node/src/Cardano/Tracing/MicroBenchmarking.hs

```diff
@@ -0,0 +1,122 @@
+{-# LANGUAGE LambdaCase             #-}
```

Comment:
Please don't align `#-}`

Reply:
Ah, sorry for aligning things.

Reply:
Wait. Why do we have it aligned in other modules then?

Reply:
@ksaric They shouldn't be aligned in other modules. Those files were most likely there before we put in `STYLE.md`

Reply:
Shouldn't someone fix that then? It's confusing to have a `STYLE.md` which doesn't have styles applied to the code.
The code will be split between two styles.
But fine, I'll remove alignment.